docs/continuous-delivery/low-code-solutions/low-code-solutions.md

```diff
@@ -0,0 +1,37 @@
+# Continuous delivery on low code solutions
+
+Low code, no code applications have taken a spot in solutions like process automation, AI models, Bots and Apps. The scenarios enabled by those platforms are constantly growing and they have left an exploration phase to take productive roles, this has been exactly the reason why bringing more professional tools to their development has became necessary.
+
+In the case of Power Platform products, the integration of a CI/CD process brings indeed more complexity to a solution oriented to [Citizen Developers](https://www.gartner.com/en/information-technology/glossary/citizen-developer) but on the other hand it strengthens the solution into a more scalable piece of software capable of dealing with new features and bug corrections in a faster way.
+
+## Environments in Power Platform Solutions
+
+Environments are the space where Power Platform Solutions exists. The store, manage and share everything related to the solution like data, apps, chatbots, flows and models. They also serve as containers to separate apps that might have different roles, security requirements or just target audiences. They will be used to create different stages of the solution development process, the expected model of working with environments in a CI/CD process will be as the following image suggests.
+
+![image](../images/environments.png)
+
+### Environments considerations
+
+Whenever an environment has been created, its resources can be only accessed by users within the same tenant. When you create an app in an environment that app can only interact with data sources that are also deployed in that same environment, this includes connections, flows and Dataverse databases. This is an important consideration when dealing with a CD process.
+
+## Deploy strategy
+
+With three environments already created the goal now is to make them interact. Each environment will require the creation of its own solution (data included).
+
+Development team will be working in **Dev** environment.
```

Comment:
As development team will work in the Dev environment, please consider adding what happens to the other environments too. This way the reader can better understand where the other environments fit in the process.